backend/app/service/runtime_utils.go

```diff
@@ -400,7 +408,7 @@ func handleCompose(env gotenv.Env, composeContent []byte, create request.Runtime
 				ports = append(ports, "${HOST_IP}:${PANEL_APP_PORT_HTTP}:${JAVA_APP_PORT}")
 			case constant.RuntimeGo:
 				ports = append(ports, "${HOST_IP}:${PANEL_APP_PORT_HTTP}:${GO_APP_PORT}")
-			case constant.RuntimePython:
+			case constant.RuntimePython, constant.RuntimeDoNet:
 				ports = append(ports, "${HOST_IP}:${PANEL_APP_PORT_HTTP}:${APP_PORT}")
 			}
 
```

Comment:
No changes needed! The code does not contain any irregularities, potential issues, or optimizations that can be suggested at this time. If there were to be any updates in the future, these would likely need to involve specific features or improvements based on new requirements or technology advancements. 

Keep in mind though, the previous knowledge cutoff is from 2021-09-01, so please update your checks as necessary when reviewing newer versions of code.